client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -94,30 +154,32 @@ const MarketplaceThankYou = ( { productSlug } ): JSX.Element => {
 	};
 
 	const thankYouSubtitle = translate( '%(pluginName)s has been installed.', {
-		args: { pluginName: plugin?.name },
+		args: { pluginName: pluginOnSite?.name },
 	} );
 
 	return (
 		<ThemeProvider theme={ theme }>
-			<Masterbar>
-				<Item
-					icon="cross"
+			<MasterbarStyled>
+				<WordPressLogoStyled />
+				<ItemStyled
+					icon="chevron-left"
 					onClick={ () =>
 						( document.location.href = `${ document.location.origin }/plugins/${ siteSlug }` )
 					} // Force reload the page.
```

Comment:
Really small edge case here when we don't have a selected site. In this case, I'm being redirected to `plugins/null`.
Not blocking as it will only happen with direct access and not "real" use case.

Reply:
yeah and should even not be possible to reach this page without a site selected, even by directly accessing you should be redirected to site selection according to 
https://github.com/Automattic/wp-calypso/blob/08961f239b9c353bb0b44983d9f7b014db038133/client/my-sites/marketplace/index.js#L64

we can address it in a follow up!